### ISSUE TITLE
Node Play Animation with local 

### ISSUE THREAD

**Issue by diegocabrera88** [IzaZed/Uchronian-Logic-UPBGE-Logic-Nodes#75]
Is it possible to include in the animation, local mode?
![image](https://user-images.githubusercontent.com/123604664/214912127-86aa5b6e-fefb-4908-9c1b-40c8faaccf84.png)

**Comment by IzaZed** [comment]
This is not possible I think. I don't know exactly what the Local option does, but I'd guess it just applies the animation on top of the current transform?
To achieve this you can animate "Delta" values instead of Loc/Rot/Scale

**Comment by diegocabrera88** [comment]
yes, but animating in delta is too tedious. I hope it can be implemented.

**Comment by BenjaminWSq** [comment]
me too
